docs/git-compatibility.md

```diff
@@ -177,6 +177,10 @@ co-located repo like so:
 # Ignore the .jj directory in Git
 echo '/*' > .jj/.gitignore
 # Move the Git repo
+if [ -d .git ] ; then
+    echo '*Warning*: Unexpected .git dir moved to .git.old'
+    mv .git .git.old
```

Comment:
Perhaps this should be just `exit 1`, so that the user can figure out what to do? You'd also need to change the message above to match.

One situation when this is better is if `.git.old` already exists.

The one downside is that, if the user is pasting the commands into a shell, this will exit the shell. Still, that might be better than doing something unexpected; hopefully they'll read the script more carefully if this happens.

Reply:
Just `set -e` and test first? I don't think the documentation example should have verbose error handling.